### PR TITLE
Add stream timeout parameters to node-cli

### PR DESCRIPTION
**Pull request by cwdsuzhou**
This PR https://github.com/virtual-kubelet/virtual-kubelet/pull/803 add stream timeout parameters, but we actually run `api.AttachPodRoutes` in node-cli, 
so we should add these parameters here, otherwise it would not effective.

This PR
1. Update go mod to make it possible to use `StreamIdleTimeout` in `api.PodHandlerConfig`
2. Expose these parameters in `node-cli`

### CODE REVIEW

**Comment by cwdsuzhou** [comment]
@cpuguy83

File: go.mod [hunk 1]
```diff
@@ -4,20 +4,28 @@ go 1.12
...
-	gotest.tools v0.0.0-20181223230014-1083505acf35
-	k8s.io/api v0.0.0-20190222213804-5cb15d344471
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+	github.com/virtual-kubelet/virtual-kubelet v1.2.1-0.20200504180557-8fc8b69d8f53
+	go.opencensus.io v0.21.0
+	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a // indirect
+	golang.org/x/text v0.3.1-0.20181227161524-e6919f6577db // indirect
+	gotest.tools v2.2.0+incompatible
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
```
Comment (cpuguy83) [inline go.mod:22]: ?

Comment (cwdsuzhou) [reply go.mod:22]: would not influent it we have used 

replace k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628

**Comment by cwdsuzhou** [comment]
seems duplicated with https://github.com/virtual-kubelet/node-cli/pull/19, I would like close it

**Pull request closed without merging**